docs/api-versioning.md

```diff
@@ -0,0 +1,52 @@
+# API Versioning for DICOM Server
+
+This guide gives an overview of the API version policies for DICOM Server. 
+
+All versions of the DICOM APIs will always conform to the DICOMweb™ Standard specifications, but versions may expose different APIs based on our conformance statment [https://github.com/microsoft/dicom-server/blob/main/docs/resources/conformance-statement.md]
+
+## Supported Versions
+
+A list of supported versions can be found in our swagger documentation [link].
+
+### Prerelease versions
+
+An API version with the label "prerelease" indicates that the version is not ready for production, and should only be used in testing environments. These endpoints may experience breaking changes without notice.
+
+### Breaking changes
+
+We will increment the API version number for any breaking changes to the API.
+
+Breaking changes:
+1. Renaming or removing endpoints
+2. Removing parameters or adding mandatory parameters
+3. Changing status code
+4. Deleting property in response or altering response type at all (but okay to add properties to the response)
+5. Changing the type of a property
+6. Behavior of an API changes (changes in buisness logic, used to do foo, now does bar)
+
+Non-breaking changes (API is not incremented):
+1. Addition of properties that are nullable or have a default value
+2. Addition of properties to a response model
+3. Changing the order of properties
+4. Introduction of paging to existing collections
```

Comment:
Is'nt this breaking? Since now my client code needs to keep calling the API for multiple pages instead of getting all results in once.

Reply:
You're right. Clients wouldn't know to look for some continuation token or some similar concept

Reply:
Thanks. I was looking at the MS Graph versioning docs https://docs.microsoft.com/en-us/graph/versioning-and-support to see what they had communicated to customers, and they had including paging as a non-breaking change. 

Now that I think about it, I agree with you both that we should count it as breaking. I have removed it from the list